cmd/registry/patch/spec.go

```diff
@@ -126,7 +129,22 @@ func applyApiSpecPatch(
 		AllowMissing: true,
 	}
 	// TODO: verify mime type
```

Comment:
Is this still a TODO?

Reply:
Because we might never do this here and rely on a checker? Yeah, I'll take this comment out.